tests/kafka/kafka.go

```diff
@@ -89,7 +89,12 @@ func main() {
 			case msg := <-breader.Messages():
 				str := msg.Binlog.String()
 				log.S().Debugf("recv: %.2000s", str)
-				ld.Input() <- loader.SlaveBinlogToTxn(msg.Binlog)
+				txn, err := loader.SlaveBinlogToTxn(msg.Binlog)
+				if err != nil {
+					log.S().Error(errors.ErrorStack(err))
+					log.S().Fatal(err)
```

Comment:
duplicated log, if you want Fatal(err) why need the previous Error log, you can just Fatale here